docs/usage/configuration-options.md

```diff
@@ -797,6 +797,20 @@ It will be compiled using Handlebars and the regex `groups` result.
 If `extractVersion` cannot be captured with a named capture group in `matchString` then it can be defined manually using this field.
 It will be compiled using Handlebars and the regex `groups` result.
 
+### matchQueries
+
+Each `matchQueries` must be a valid regular expression, optionally with named capture groups.
```

Comment:
```suggestion
Each `matchQueries` must be a valid jsonata query (escaped).
```

And does it need or support more than one?

Reply:
Forgot to push this change 😅

Reply:
Yes, it supports more than one